src/actions/approveDomainTransfer.ts

```diff
@@ -0,0 +1,17 @@
+import { ethers } from "ethers";
+import { getERC721Contract } from "../contracts";
+
+// Approve zAuction to transfer domains on behalf of the signer
+export const approveDomainTransfer = async (
```

Comment:
I don't see this being used anywhere

Reply:
It is used in `approveZAuctionTransferNftByBid` and `approveZAuctionTransferNft`